syntax = "proto3";

import "taskmap.proto";
import "database.proto";

service JokeRetriever{
    rpc get_random_joke(Void) returns (ExtraInfo) {}
}